index.js

```diff
@@ -69,31 +71,93 @@ const customerReducer = combineActionHandlers(Set(), {
   [removeCustomer]: (state, { customer }) => state.delete(customer)
 })
 
-const store = createStore(combineReducers({
-  books: bookReducer,
-  customers: customerReducer
-}))
+const receiveLogin = createActionCreator(
+    'RECEIVE_LOGIN',
```

Comment:
IMHO this action should be `login`, symmetrically to `logout`.

Otherwise good fixes 👍